src/index.ts

```diff
@@ -166,32 +186,49 @@ export const formatValidationError = (error: t.ValidationError) =>
  * @category formatters
  * @since 1.2.0
  */
-export const formatValidationErrors = (errors: t.Errors) =>
+export const formatValidationErrors = (
+  errors: t.Errors,
+  options?: ReporterOptions
+) =>
   pipe(
     errors,
     groupByKey,
-    R.mapWithIndex(format),
+    R.mapWithIndex((path, errors) => format(path, errors, options)),
     R.compact,
     R.toArray,
     A.map(([_key, error]) => error)
   );
 
+export interface ReporterOptions {
```

Comment:
This should fix the CI failure

```suggestion
/**
 * @category formatters
 * @since 1.2.2
 */
export interface ReporterOptions {
```